### ISSUE TITLE
ERROR - Android 3.2 and older - AbstractMethodError: abstract method not implemented

### ISSUE THREAD

**Issue by jkasten2** [OneSignal/OneSignal-Android-SDK#157]
Crash happens on some pre-Android 4.0 devices.

```Java
Fatal Exception: java.lang.AbstractMethodError: abstract method not implemented 
at com.onesignal.ActivityLifecycleListenerCompat$1$1.onPaused(ActivityLifecycleListenerCompat.java) 
at android.app.ActivityThread.performPauseActivity(ActivityThread.java:2373) 
at android.app.ActivityThread.performPauseActivity(ActivityThread.java:2315) 
at android.app.ActivityThread.handlePauseActivity(ActivityThread.java:2295) 
at android.app.ActivityThread.access$1700(ActivityThread.java:117) 
at android.app.ActivityThread$H.handleMessage(ActivityThread.java:942) 
at android.os.Handler.dispatchMessage(Handler.java:99) 
at android.os.Looper.loop(Looper.java:130) 
at android.app.ActivityThread.main(ActivityThread.java:3687) 
at java.lang.reflect.Method.invokeNative(Method.java) 
at java.lang.reflect.Method.invoke(Method.java:507) 
at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:867) 
at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:625) 
at dalvik.system.NativeStart.main(NativeStart.java)
```

**Comment by planethcom** [comment]
Is there a way to get rid of that error?
It actually only happens in the release builds, NOT in debug builds. 
Could it be that there's something wrong in the proguard rules?

**Comment by jkasten2** [comment]
@planethcom Fixed in commit 4a18070dd007ddddc3cb81242a3b34ab152637e8 and released in version 3.4.3.
